easybuild/tools/py2vs3/py2.py

```diff
@@ -63,9 +57,12 @@ def subprocess_popen_text(cmd, **kwargs):
     return subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs)
 
 
+# Wrapped in exec to avoid invalid syntax warnings for Python 3
```

Comment:
Interesting approach to work around this invalid syntax error when installing EasyBuild with Python 3...

Maybe the comment should be clarified a bit though: the invalid syntax issue only occurs when this module is loaded when EasyBuild is installed with Python 3, and it's perfectly fine to ignore that because when running EasyBuild with Python 3, the `tools.py2vs3.py2` module will never be imported.

Reply:
I wouldn't spend to many letters on that corner case. I mean it is in "py2.py", the comment says "warnings" and "Python 3"
IMO that says enough already :)